fetch.bs

```diff
@@ -728,8 +728,9 @@ explicitly set <a for=/>request</a>'s
 this flag set are subject to additional processing requirements.
 
 <p>A <a for=/>request</a> has an associated
-<dfn export>skip-service-worker flag</dfn>. Unless stated otherwise it is
-unset.
+<dfn export>use-service-workers value</dfn>, that is "<code>all</code>",
```

Comment:
What do you think about `<dfn export for=request>service-workers mode</dfn>`? I think that would be more consistent with what we have thus far.